reconcile/utils/oc.py

```diff
@@ -63,6 +64,55 @@ class JobNotRunningError(Exception):
 
 
 class OC:
+    class _Decorators:
+        @classmethod
+        def process_reconcile_time(cls, function):
+
+            @wraps(function)
+            def wrapper(*args, **kwargs):
+                result = function(*args, **kwargs)
+
+                running_state = RunningState()
+                commit_time = float(running_state.timestamp)
+                time_spent = time.time() - commit_time
+
+                try:
+                    kind = result['resource']['kind']
+                    name = result['resource']['metadata']['name']
+                    annotations = \
+                        result['resource']['metadata'].get('annotations')
+                except KeyError as e:
+                    logging.warning(f"Error processing metric: {e}")
+                    return
+
+                ignore_reconcile_time = \
+                    annotations.get('qontract.ignore_reconcile_time') == 'true'
```

Comment:
`annotations` can be None.

Reply:
indeed, I somehow thought this was going to work testing the wrong thing.

Reply:
Fixed in 69576f7